sunpy/instr/goes.py

```diff
@@ -88,31 +145,35 @@ def get_goes_event_list(timerange, goes_class_filter=None):
 
     return goes_event_list
 
-def temp_em(goeslc, abundances="coronal", download=False, download_dir=DATA_PATH):
+def calculate_temperature_em(goeslc, abundances="coronal",
+                             download=False, download_dir=None):
```

Comment:
I may have missed this out... but, should not we default to `DATA_PATH` for downloading? that would remove the `if` in l.245, no?